src/TFR.jl

```diff
@@ -18,7 +18,7 @@ function spectrogram(audio::SampleBuf{T, 2},
                         hopsize::Int = windowsize >> 2;
                         window = hanning, kwargs...) where T
     noverlap = windowsize - hopsize
-    (mapslices(data, 1) do data
+    (mapslices(audio.data, dims=1) do data
         DSP.spectrogram(data, windowsize, noverlap; fs = audio.samplerate, window = window, kwargs...)
     end)[:]
```

Comment:
I've noticed here that instead of putting `[:]` at the end and making an unecessary copy, we can wrap the entire thing in a `vec(...)` function.

Reply:
I assume this applies for all instances of `[:]` ? If so I'll change them all I as I go